### PR TITLE
POLCT-203: Pass Jira options to prescription in IO stage

### PR DESCRIPTION
**Pull request by sig-chakrava**

### CODE REVIEW

**Review by sig-suhaslr — APPROVED** [review]
LGTM

File: io-template.yml [hunk 1]
```diff
@@ -51,10 +51,18 @@ IO:
...
         --coverity.username=$COVERITY_USERNAME 
-        --coverity.password=$COVERITY_PASSWORD
+        --coverity.password=$COVERITY_PASSWORD 
+        --jira.api.url=$JIRA_URL 
+        --jira.project.name=$JIRA_PROJECT_KEY 
+        --jira.issues.query=$JIRA_ISSUES_QUERY 
+        --jira.username=$JIRA_USERNAME 
+        --jira.auth.token=$JIRA_AUTH_TOKEN 
+        --jira.assignee=$JIRA_ASSIGNEE 
+        --jira.enable=$ENABLE_JIRA
```
Comment (sig-suhaslr) [inline io-template.yml:61]: Can we change ENABLE_JIRA to JIRA_ENABLE to make it consistent with other variables?

**Review by sig-suhaslr — CHANGES_REQUESTED** [review]

File: io-template.yml (hunk 1 above)
Comment (sig-chakrava) [reply io-template.yml:61]: ENABLE_JIRA looks better to me as its consistent with English grammar, even the default placeholder value is  <<ENABLE_JIRA>> in default io-manifest.yml - https://github.com/synopsys-sig/io-artifacts/blob/main/io-manifest.yml#L29

**Review by sig-suhaslr — CHANGES_REQUESTED** [review]

File: io-template.yml (hunk 1 above)
Comment (sig-suhaslr) [reply io-template.yml:61]: In my view a variable name should be consistent with the variable names defined in the same file. In here ENABLE_JIRA is not consistent with the variables names defined in io-template.yml. If you think the new variable name is correct then i'm fine with it

**Review by rahulguna10 — APPROVED** [review]

**Review by sig-suhaslr — APPROVED** [review]

**Pull request merged**